lib/gui/app.js

```diff
@@ -160,23 +160,25 @@ app.run(() => {
   store.subscribe(() => {
     const currentFlashState = flashState.getFlashState();
 
+    if (!flashState.isFlashing()) {
+      return;
+    }
+
     // There is usually a short time period between the `isFlashing()`
     // property being set, and the flashing actually starting, which
     // might cause some non-sense flashing state logs including
     // `undefined` values.
     //
     // We use the presence of `.eta` to determine that the actual
     // writing started.
```

Comment:
We need to delve into `child-writer` to get accurate results for the "Elevating..." label, which unfortunately lives in another process so I've been setting up `electron-mocha` hoping that will help.